File: Sort Algorithms/bubble.py
def bubbleSort(array):
	arrayLength = len(array)
	for i in range(arrayLength, 1, -1):
		for j in range(0, i-1):
			if (array[j] > array[j+1]):
				temp = array[j]
				array[j] = array[j+1]
				array[j+1] = temp
	return array

example1 = [3,2]
example2 = [9,3,7,4,6]

result = bubbleSort(example1)
print(result)

File: Sort Algorithms/bubble.js
//Bubble Sort
const bubbleSort = (list) => {
	const swapFunction = (list, i, j) => {
		[list[i], list[j]] = [list[j], list[i]]
	}
	for (let i = list.length; i > 1; i--) {
		for (let j = 0; j < i -1; j++) {
			if (list[j] > list[j+1]) {
				swapFunction(list, j, j+1)
			}
		}
	}
	return list
}

let example1 = [3,2]
let example2 = [5,7,4,9,1,8]
let result = bubbleSort(example1)
console.log(result)